integration_tests/__tests__/transform-test.js

```diff
@@ -98,7 +98,7 @@ describe('multiple-transformers', () => {
   });
 
   it('transforms dependencies using specific transformers', () => {
-    const {json, stderr} = runJest.json(dir, ['--no-cache']);
+    const {json, stderr} = runJest.json(dir, ['--no-cache', '-u']);
```

Comment:
Oh I know what happened, I've updated it manually before this change https://github.com/facebook/jest/pull/2896#discussion_r101373197, so it wasn't updated. You're totally right and fix is already applied, thanks!